pkgs/applications/graphics/gscreenshot/default.nix

```diff
@@ -0,0 +1,79 @@
+{ lib
+, fetchFromGitHub
+, buildPythonApplication
+, gettext
+, gobject-introspection
+, gtk3
+, pillow
+, pygobject3
+, scrot
+, setuptools
+, slop
+, wrapGAppsHook
+, xlib
+, xclip ? null
+, xdg-utils ? null
+}:
+
+buildPythonApplication rec {
+  pname = "gscreenshot";
+  version = "3.3.0";
+
+  src = fetchFromGitHub {
+    owner = "thenaterhood";
+    repo = "${pname}";
+    rev = "v${version}";
+    sha256 = "vAJyCamAI8N2InbMYvtzeuPSLkWT3+xA0yyS2Bp5alw=";
+  };
+
+  strictDeps = false;
+  doCheck = false;
```

Comment:
Are those necessary?

Reply:
They are, the build fails without either of them. The tests run when `doCheck` is enabled require a display and therefore fail. Regarding `strictDeps`, see the warning in [this section of the Nixpkgs manual](https://nixos.org/manual/nixpkgs/stable/#ssec-gnome-hooks); `wrapGAppsHook` doesn't function without it.

I borrowed these from other packages which I can no longer find, as the repository's search functionality no longer seems to return any results for code.